Ronin.rte/Devices/Special/Scrambler/Scrambler.ini

```diff
@@ -76,13 +93,13 @@ AddDevice = MOSRotating
 		CopyOf = Null AtomGroup
 	GibSound = SoundContainer
 		AddSound = ContentFile
-			FilePath = Ronin.rte/Devices/Special/Scrambler/Sounds/Explosion01.flac
+			FilePath = Ronin.rte/Devices/Special/Scrambler/Sounds/Explosion1.flac
 		AddSound = ContentFile
-			FilePath = Ronin.rte/Devices/Special/Scrambler/Sounds/Explosion02.flac
+			FilePath = Ronin.rte/Devices/Special/Scrambler/Sounds/Explosion2.flac
 		AddSound = ContentFile
-			FilePath = Ronin.rte/Devices/Special/Scrambler/Sounds/Explosion01.flac
+			FilePath = Ronin.rte/Devices/Special/Scrambler/Sounds/Explosion1.flac
```

Comment:
Why twice?

Reply:
Because for whatever reason fmod is locked to alternate between two sounds in one SoundContainer rather than picking one at random, which it does at 3 sounds and more.

This gib SoundContainer would only ever play the first sound on every occasion.

Reply:
Maybe?
@garethyr

Reply:
Haha this was a derp on my part I guess. I basically did what sprite frames do - i.e. just cycle back and forth if you only have 2 options, cause it's functionally the same, but I didn't think about this case where it only plays once.
Probably the best way to solve this is to remove SoundSet.cpp lines 276 - 278, so there's no Case 2 handling.

Also, for future reference, all the in-depth SoundContainer logic and modes and stuff is generally our code, not FMOD. FMOD mostly just worries about playing sounds good.